### PR TITLE
A user can add an item to their cart.

### PR DESCRIPTION
**Pull request by ChrisJVoss**
screen shot 2017-06-19 at 2 28 22 pm

screen shot 2017-06-19 at 2 28 55 pm

screen shot 2017-06-19 at 2 29 18 pm

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -73,7 +75,8 @@ function displayItemDetails(itemList, productId, itemContainer, row) {
   var id = getView(views, $id)
   swapToView(views[id], views)
   itemDescription(itemList, productId, product)
-  createReturnButton(product)
+  // createReturnButton(product)
```
Comment (victormejia) [inline main.js]: if this is no longer needed, let's get rid of it

File: main.js [hunk 2]
```diff
@@ -124,3 +128,27 @@ function getView(views, id) {
...
+  })
+}
+
+function addToCart(product, cart, itemList) {
+  var cartId = product.id
+  cart.unshift(itemList[cartId]);
+}
+
+function updateCartButton(cartSize) {
+  count = cartSize.length
```
Comment (victormejia) [inline main.js]: don't forget to declare `var count`, otherwise you're declaring a global variable

File: main.js [hunk 3]
```diff
@@ -124,3 +128,27 @@ function getView(views, id) {
...
+  product.appendChild(addToCartButton)
+  addToCartButton.addEventListener('click', function(event){
+    addToCart(product, shoppingCart, itemList)
+    updateCartButton(shoppingCart)
+  })
+}
+
+function addToCart(product, cart, itemList) {
+  var cartId = product.id
+  cart.unshift(itemList[cartId]);
```
Comment (victormejia) [inline main.js]: is this to have the latest items added to the front?

**Review by victormejia — CHANGES_REQUESTED** [review]
great job so far

File: main.js (hunk 3 above)
Comment (ChrisJVoss) [reply main.js]: Yea, it made sense to me that the newest items would come first? Like they are at the top of the shopping cart. Should I change it to push?

**Review by victormejia — APPROVED** [review]
great job!

**Pull request merged**